homeassistant/components/esphome/bluetooth/client.py

```diff
@@ -185,7 +169,7 @@ def __init__(
 
     def __str__(self) -> str:
         """Return the string representation of the client."""
-        return f"ESPHomeClient ({self.address})"
+        return f"ESPHomeClient ({self._description})"
 
     def _unsubscribe_connection_state(self) -> None:
```

Comment:
The try here likely isn't needed here anymore as it should be unreachable but there are no test in the library for that yet so not this PR